packages/compiler/test/server/signature-help.test.ts

```diff
@@ -1,5 +1,5 @@
 import { deepStrictEqual } from "assert";
-import { SignatureHelp } from "vscode-languageserver/node.js";
+import { MarkupKind, SignatureHelp } from "vscode-languageserver/node.js";
 import { createTestServerHost, extractCursor } from "../../testing/test-server-host.js";
 
 describe("compiler: server: signature help", () => {
```

Comment:
Trying this and I don't get the doc after the params

<img width="553" alt="image" src="https://user-images.githubusercontent.com/1031227/205157732-064a5255-ff95-4b4a-ab2e-1421443ee216.png">
https://cadlplayground.z22.web.core.windows.net/prs/1346/?c=LyoqCiAqIFRoaXMgaXMgYSBmb28gZG9jxBXEBEBwYXJhbSBuYW1lIHTJJcQPySJleGFtcGzJEGBgYGNhZGzFG2ZvbygiTXnELyIsIDEyKcQWbW9kZWwgQmFyIHt9xzEKICovCmV4dGVybiBkZWPkAIUodGFyZ2V0OiB1bmtub3duLMV1OiBzdHJpbmcsIG90aGVyOiBpbnQzMik6IHZvaWQ7CgoKCsV0KQrGZUZvbyB7fQ%3D%3D

Reply:
Other issue if you have a decorator outside of a namespace 
<img width="504" alt="image" src="https://user-images.githubusercontent.com/1031227/205157868-64ec9c1e-a8b1-4520-89d9-ec5ba6672808.png">

Reply:
hhm so 1st problem is because of `@example` is that a tag we decided should exists? I guess I wrote the docs using that in https://github.com/microsoft/cadl/pull/1340

Reply:
I now get this with my latest push:

![image](https://user-images.githubusercontent.com/75470/205160627-9aa374c9-d349-4d8a-a19a-642319516122.png)

Reply:
But playground may have other quirks, so will try there too.

Reply:
link above seems to work (kinda, just not splitting lines) 
<img width="812" alt="image" src="https://user-images.githubusercontent.com/1031227/205161019-3e292666-0c1f-40fd-85b2-6a6c63c5cb9e.png">

Reply:
actually does work when you hover but not when you autocomplete

Reply:
<img width="332" alt="image" src="https://user-images.githubusercontent.com/1031227/205161435-56779351-de63-485e-9de6-9b962e547444.png">
getting this when doing the same in TS

Reply:
Hmm, I could swear I tested TS and it didn't show the tags on signature-help so I mimicked.

Reply:
Oh, it only hides the param tags, I guess!

Reply:
though seems like the postdoc section gets lost in TS too

Reply:
Note that this is post doc is part of param b here.

Reply:
is the rule anything that comes after a tag gets picked up as the tag including new lines? or it stops at new line

Reply:
guess you just answer that question :)

Reply:
Things should be better now with the last push.

Reply:
Try hovering over that entry then clicking on the right arrow that appears. VS Code and Monaco seem to remember if you did that and then keep showing you details, but playground will reset to hide details by default on reload. I'm pretty sure nothing changed here and I can see the same thing if I do @doc on the main playground without these changes where I won't see the signature until I click the right arrow.

Reply:
yeah that was it, then it all works great!